### PR TITLE
Contributer guide

### PR DESCRIPTION
**Pull request by jguiditta**
As a contributer I would like a guide to explain the process of contributing to Tim

### CODE REVIEW

**Comment by jguiditta** [comment]
Fyi, I did notice a couple formatting issues, but haven't been able to figure them out.  I am wondering if they are something I actually did wrong, or something to do with how github renders rdoc (and if the latter, should we change this to be markdown?).  The items that have incorrect formatting are:
- blank line inserted between bullet points
- Using asterisks around words did not set them to _bold_ as I expected
- Using underlines did not set _italic_ as expected.

**Comment by jguiditta** [comment]
Hmph, well at least for the bullet list, rdoc _itself_ has the same formatting issue: https://github.com/rdoc/rdoc/blob/master/TODO.rdoc

**Comment by jguiditta** [comment]
Ok, updated with html tags for underline and bold, which seems to render properly.  I am seeing a lot of rdoc READMEs that have the blank lines in bullet lists, haven't found one that looks correct yet, so we may have to just ignore that bit for now.

File: README.rdoc [hunk 1]
```diff
@@ -60,6 +60,59 @@ From test/dummy run:
...
+implementing it.  Ideally this would be in a format focusing on:
+
+* Who would be the user of this feature?
+* How would the feature be used (this could be a workflow, api
+  example, etc)?
+* What need does it fill (in other words, why does the user want/need
+  to do the described task)?
+
+If you like, you may use the Connextra format, but it is not strictly
+required.  For reference, that format is:
```
Comment (mtaylor) [inline README.rdoc:97]: Can you put a link to the Connextra format here.  

I think just User Story format might be better than referencing Connextra.  http://en.wikipedia.org/wiki/User_story

**Comment by mtaylor** [comment]
I am going to move this and all wiki pages into a docs folder as you suggested here: https://github.com/aeolus-incubator/tim/issues/120

I will merge this pull request and make the changes in the new docs pull request.

**Pull request merged**